manticore/core/cpu/disasm.py

```diff
@@ -141,6 +138,18 @@ def unimplemented(self, il):
         return (il.operation == enums.LowLevelILOperation.LLIL_UNIMPL or
                 il.operation == enums.LowLevelILOperation.LLIL_UNIMPL_MEM)
 
+    def _get_current_llil_func(self, code, pc):
+        if pc in self.llil_func_cache:
+            return self.llil_func_cache[pc]
+
+        from binaryninja import Architecture, LowLevelILFunction
+        # FIXME
+        func = LowLevelILFunction(Architecture['x86_64'])
+        func.current_address = pc
+        size = self.view.arch.get_instruction_low_level_il(code, pc, func)
+        self.llil_func_cache[pc] = (func, size)
+        return func, size
```

Comment:
It might be counterintuitive to return both `func` and `size` from a function called `_get_current_llil_func`, we should consider either splitting the function in two or renaming it